.changeset/config.json

```diff
@@ -0,0 +1,12 @@
+{
+  "$schema": "https://unpkg.com/@changesets/config@3.0.0/schema.json",
+  "changelog": "@changesets/cli/changelog",
+  "commit": false,
+  "fixed": [],
+  "linked": [],
+  "access": "public",
```

Comment:
What does `access: public` mean in this case?

Reply:
It sets how packages are published, and it defaults to `restricted` since npm does. More info here: https://github.com/changesets/changesets/blob/main/docs/config-file-options.md#access-restricted--public